project/legislation/propublica/congress/bills.py

```diff
@@ -0,0 +1,69 @@
+from .client import Client
+from .utils import CURRENT_CONGRESS, check_chamber
+
+
+class BillsClient(Client):
+
+    def by_member(self, member_id, type='introduced'):
+        """
+        Takes a bioguide ID and a type:
+        (introduced|updated|cosponsored|withdrawn)
+        Returns recent bills
+        """
+        path = "members/{member_id}/bills/{type}.json".format(
+            member_id=member_id, type=type)
+        return self.fetch(path)
+
+    def get(self, bill_id, congress=CURRENT_CONGRESS, type=None):
+        if type:
+            path = "{congress}/bills/{bill_id}/{type}.json".format(
```

Comment:
[Use f-string](https://realpython.com/python-f-strings/) here, rather than 'str.format()`. F-strings are easier to read, for situations like this.